dwave/cloud/utils.py

```diff
@@ -303,6 +304,74 @@ def create_url(self, url):
         return urljoin(self.base_url, url)
 
 
+def hasinstance(iterable, class_or_tuple):
```

Comment:
I am sure there is a reason, but I am confused why this and `hasexception` need to be functions rather than just one-liners? It looks like you only use it once unless I missed something.

Reply:
You're right, it's used just once now, but it the earlier versions it was used in multiple places. I still think it might be useful to keep it in utils.